addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -71,18 +71,20 @@ func process_authority():
 	
 	# Gather state properties that we own
 	# i.e. it's the state of a node that belongs to the local peer
-	for property in state_properties:
-		var pe = _property_cache.get_entry(property)
-		if pe.node.is_multiplayer_authority():
-			_auth_state_props.push_back(pe)
+	var picked_property_entry: PropertyEntry
+	for picked_property in state_properties:
+		picked_property_entry = _property_cache.get_entry(picked_property)
+		if picked_property_entry.node.is_multiplayer_authority():
+			_auth_state_props.push_back(picked_property_entry)
 
 	# Gather input properties that we own
 	# Only record input that is our own
-	for property in input_properties:
-		var pe = _property_cache.get_entry(property)
-		if pe.node.is_multiplayer_authority():
-			_record_input_props.push_back(pe)
-			_auth_input_props.push_back(pe)
+	for picked_property in input_properties:
+		picked_property_entry = _property_cache.get_entry(picked_property)
+		_record_input_props.push_back(picked_property_entry)
```

Comment:
Note that `_record_input_props` is set here *outside* authority. Solves #250 and its tested to work locally, as intended in all my other PRs. After all, it is not set anywhere if not here, and before this commit, `_record_input_props` is the same as `_auth_input_props` for exclusively the authority player.